release.js

```diff
@@ -107,15 +107,13 @@ async function release() {
 
     const releaseOptions = {
       increment: version,
+      requireCleanWorkingDir: false,
       npm: {
         publish: false,
       },
-      github: {
-        release: false,
-      },
       prompt: {
         src: {
-          release: false,
+          release: true,
```

Comment:
What does this do?

Reply:
It'll prompt with "are you ready to release" before tagging on github, I originally turned it off but I accidentally pushed tags to github while testing, so it's for preventing dangerous individual like myself to publish while testing 😄

Reply:
👍